indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -123,10 +124,40 @@ NoResults.defaultProps = {
   query: undefined,
 };
 
+function SearchOptions({sort, sortOptions, onSortChange}) {
+  const selected = sortOptions.find(x => x.key === sort) || sortOptions[0];
+
+  return !sortOptions.length ? null : (
```

Comment:
IMO this would be nicer:

```js
if (!sortOptions.length) {
  return null;
}

return ...;
```

Reply:
It's funny because I changed that shortly before pushing 😄